builtin/credential/ldap/backend.go

```diff
@@ -206,13 +154,11 @@ func (b *backend) Login(ctx context.Context, req *logical.Request, username stri
 		return username, policies, ldapResponse, allGroups, nil
 	}
 
-	entityAliasAttribute, err := ldapClient.GetUserAliasAttributeValue(cfg.ConfigEntry, c, username)
-	if err != nil {
-		return "", nil, logical.ErrorResponse(err.Error()), nil, nil
-	}
-	if entityAliasAttribute == "" {
+	cn := c.UserAttributes["cn"]
```

Comment:
I'm not sure this is quite correct.  It appears that
`Client.GetUserAliasAttributeValue(...)` retrieves `cfg.UserAttr` which could be
really any attribute name (of course it's typically `cn` or `uid`).  So perhaps
this should be something like `userAttrValues := c.UserAttributes[cfg.UserAttr]`?

Reply:
You're right, thanks for catching this!